### PR TITLE
[import wizard] Set up the skeleton for V2

### PR DESCRIPTION
**Pull request by chejennifer**
- added the changes to have V2 appear at /import/new
- types.ts is copied from V1 - static/js/import2 will have all the files needed for V2 (even if they're duplicate files from V1) so that when we do the flip, we can just rename the folder and delete the V1 folder

### CODE REVIEW

File: server/templates/import_wizard2.html [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+{% set subpage_title = "Import Wizard" %}
+{% set subpage_url = url_for('import_wizard.main_new') %}
+{% set title = "Import Wizard" %}
+{% set main_id = 'main-import-wizard' %}
+{% set page_id = 'page-import-wizard' %}
+
+{% block head %}
+  <link rel="stylesheet" href={{url_for('static', filename='css/import_wizard2.min.css')}} >
+  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
+  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons+Outlined">
```
Comment (beets) [inline server/templates/import_wizard2.html:28]: nit: pull these in only if required.

File: static/js/import_wizard2/import_wizard.ts [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+ */
+import React from "react";
+import ReactDOM from "react-dom";
+
+import { Page } from "./components/page";
+
+ReactDOM.render(
+  React.createElement(Page),
+  document.getElementById("main-pane")
+);
```
Comment (beets) [inline static/js/import_wizard2/import_wizard.ts:24]: this should be set as a callback to `window.onload`

**Review by beets — APPROVED** [review]
only small nits!

File: server/templates/import_wizard2.html (hunk 1 above)
Comment (chejennifer) [reply server/templates/import_wizard2.html]: done

File: static/js/import_wizard2/import_wizard.ts (hunk 2 above)
Comment (chejennifer) [reply static/js/import_wizard2/import_wizard.ts]: done

**Pull request merged**